tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -44,3 +44,24 @@ def test_basic_scene_l_flag(tmp_path, manim_cfg_file, simple_scenes_path):
     ]
     out, err, exit_code = capture(command)
     assert exit_code == 0, err
+
+
+@pytest.mark.slow
+@video_comparison(
+    "SceneWithMultipleCallsWithNFlag.json",
+    "videos/simple_scenes/1080p60/SceneWithMultipleCalls.mp4",
+)
+def test_n_flag(tmp_path, simple_scenes_path):
+    scene_name = "SceneWithMultipleCalls"
+    command = [
+        "python",
+        "-m",
+        "manim",
+        simple_scenes_path,
+        scene_name,
+        "-n 3,6",
+        "--media_dir",
+        str(tmp_path),
+    ]
+    _, err, exit_code = capture(command)
+    assert exit_code == 0, err
```

Comment:
```suggestion
    assert exit_code == 0, err

```

Just added a newline at the end.